inputs/templates/test/GATKSVPipelineSingleSample/GATKSVPipelineSingleSample.no_melt.json.tmpl

```diff
@@ -27,19 +27,19 @@
   "GATKSVPipelineSingleSample.cnmops_docker": {{ dockers.cnmops_docker | tojson }},
   "GATKSVPipelineSingleSample.condense_counts_docker" : {{ dockers.condense_counts_docker | tojson }},
   "GATKSVPipelineSingleSample.gatk_docker": {{ dockers.gatk_docker | tojson }},
-  "GATKSVPipelineSingleSample.gatk_docker_pesr_override" : {{ dockers.gatk_docker | tojson }},
+  "GATKSVPipelineSingleSample.gatk_docker" : {{ dockers.gatk_docker | tojson }},
   "GATKSVPipelineSingleSample.genomes_in_the_cloud_docker" : {{ dockers.genomes_in_the_cloud_docker | tojson }},
 
   "GATKSVPipelineSingleSample.linux_docker" : {{ dockers.linux_docker | tojson }},
   "GATKSVPipelineSingleSample.manta_docker": {{ dockers.manta_docker | tojson }},
   "GATKSVPipelineSingleSample.sv_base_docker": {{ dockers.sv_base_docker | tojson }},
-  "GATKSVPipelineSingleSample.sv_pipeline_base_docker": {{ dockers.sv_pipeline_base_docker | tojson }},
+  "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
   "GATKSVPipelineSingleSample.sv_base_mini_docker": {{ dockers.sv_base_mini_docker | tojson }},
   "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
-  "GATKSVPipelineSingleSample.sv_pipeline_hail_docker": {{ dockers.sv_pipeline_hail_docker | tojson }},
-  "GATKSVPipelineSingleSample.sv_pipeline_updates_docker": {{ dockers.sv_pipeline_updates_docker | tojson }},
-  "GATKSVPipelineSingleSample.sv_pipeline_qc_docker": {{ dockers.sv_pipeline_qc_docker | tojson }},
-  "GATKSVPipelineSingleSample.sv_pipeline_rdtest_docker": {{ dockers.sv_pipeline_rdtest_docker | tojson }},
+  "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
+  "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
+  "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
+  "GATKSVPipelineSingleSample.sv_pipeline_docker": {{ dockers.sv_pipeline_docker | tojson }},
```

Comment:
For the first pass, I would keep workflow variables unchanged (e.g., `GATKSVPipelineSingleSample.sv_pipeline_hail_docker"`) as it could simplify tracking changes.